src/app/components/sandbox/CodeEditor/index.js

```diff
@@ -136,6 +136,8 @@ export default class CodeEditor extends React.PureComponent {
       if (this.props.preferences !== prevProps.preferences) {
         this.setCodeMirrorPreferences();
       }
+
+      this.handleEmmet();
```

Comment:
Small nitpick: Maybe this should be renamed to a more descriptive name like `configureEmmet`.